### PR TITLE
added fun fact

### PR DESCRIPTION
**Pull request by darrylw330**
Added something about me

### CODE REVIEW

**Review by johnLeavell — APPROVED** [review]
Looks good

File: README.md [hunk 1]
```diff
@@ -20,4 +20,4 @@ Clone this repository (or open in codespace), create a branch with your name, ad
 ## 2023-2.1 Fullstack Pancakes
 
 | Name | Fun Fact |
-| --- | --- |
```
Comment (heratyian) [inline README.md:23]: We want to keep this line to separate header from rows

File: README.md [hunk 2]
```diff
@@ -24,4 +24,4 @@ Clone this repository (or open in codespace), create a branch with your name, ad
 
 | Name | Fun Fact |
 | --- | --- |
-| Dustin Doyle | Australia is wider than the moon. |
```
Comment (heratyian) [inline README.md:27]: Seems like you deleted @Dantexkilljoy fun fact when resolving merge conflict

**Review by heratyian — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (darrylw330) [reply README.md:27]: My apologies I'll resolve

Comment (heratyian) [reply README.md:27]: looks like you still have some leftovers on line 28 and 30

**Review by heratyian — CHANGES_REQUESTED** [review]

**Pull request merged**